vagrant-all.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -eo pipefail
+
+export KUBESPAWN_AUTOBUILD="true"
+export KUBESPAWN_REDIRECT_TRAFFIC="true"
+vagrant up
```

Comment:
I would assume that it runs with fedora, as `vagrant up` without machine name defaults to fedora.